src/pybind/mgr/dashboard_v2/frontend/package.json

```diff
@@ -55,7 +55,8 @@
     "karma-jasmine-html-reporter": "^0.2.2",
     "karma-junit-reporter": "^1.2.0",
     "karma-phantomjs-launcher": "^1.0.4",
-    "node": "^8.9.4",
```

Comment:
Next version of Angular, which should be release by the end of this month, will require node 8+.
I would prefer if we could find a solution that did not prevent us from upgrading Angular.

Since this is causing problems in a few distros, maybe we could remove the dependency from package.json and add a version validation in package.json and/or in install-deps.sh script. That way the user would know what the exact problem is and could manually update his node version.